### ISSUE TITLE
Offline Transactions not working

### ISSUE THREAD

**Issue by federicoon** [kvhnuke/etherwallet#180]
I tried with both Firefox and Chrome: on step 1 I type in a From Address, which is accepted as valid, but then clicking on "Generate information" has no effect: nothing happens, no info is displayed!

Furthermore, step 2 on an offline computer hangs with Firefox on wallet decryption. It only works with Chrome (but I still miss the information which should have been generated on step 1).

Anyway, I can't set ETC only/ETH only as with online transactions. How can that be done with offline transactions?

**Comment by tayvano** [comment]
For step 1, are you doing this on an online computer? I'm having no issues in Chrome or Firefox.

Firefox has trouble decrypting Mist-encrypted Keystore files due to the fact that firefox has a shorter timeout period and javascript is not an efficient way to decrypt something that has been hashed like a million times.

If you prefer using Firefox, open your wallet in Chrome using the "View Wallet Info" tab and download another encrypted version of your private key and use that moving forward for MyEtherWallet.com. 

If you want to send via the contract via the offline tab, you will need to manually do it. That means inputting the SafeSplit Contract's address and generating the data it requires. I do not recommend this unless you know what you are doing. 

If you want to split your ETH/ETC, I recommend making 2 new fresh cold wallets, bringing your offline wallet online, transferring to your new wallets, and using standard transactions in ETH and ETC moving forward. The bottom of this post details the process fairly well: https://steemit.com/ethereum/@ktmgen/eli5-understanding-wallets-in-eth-and-etc-a-comment-from-reddit

**Comment by federicoon** [comment]
For step 1 I confirm it was from an online computer. After some more troubleshooting, I realized the issue occurs whenever I set a http proxy. It works well instead in case of direct access to internet.

Then I don't understand what you mean by "download another encrypted version of your private key", sorry.
So you're suggesting to split ETH/ETC balance via online transactions only? Ok, I'll consider that option, thanks.

**Comment by tayvano** [comment]
Okay that http proxy is also and issue that we are having showing balances in some of these. I'll note this there as well. 

The "download another version of the private key" - basically if you go to the View Wallet Details tab you can download a file that still be your keystore file and will work in Mist. But its hashed slightly fewer times and the javascript has an easier way to decrypt it so it won't time out, etc. in Firefox.

**Comment by tayvano** [comment]
HTTP proxy issue was fixed

Still aware of slow decrypt times for Mist wallets in FF and it's on the list.

Users will need to know how to calculate data required for the only ETC / only ETH transactions via contract to use the offline tab for now.

**Comment by federicoon** [comment]
I just tried to create an offline transaction, but the "Generate information" button is still not working.
Still using a proxy.

**Comment by federicoon** [comment]
Yet another failure in step 3 (from the online computer) of an offline transaction with a HTTP proxy.
It didn't go through with Firefox, it worked with Chrome only

**Comment by arthurhenning** [comment]
Any updates on this issue? Still not working on an airgapped computer. Tried various bowsers, versions and private modes, with no luck.

**Comment by Ligrogen** [comment]
I had the problem like this. The "Generate Transaction" button was not enabled. I wasted lot of time solving this issue. Finally I had found a very simple solution - I had changed the screen resolution at my offline laptop from 800*600 to 1024*768 and the "Generate Transaction" button became enabled :)
